nixos/tests/mysql/mariadb-galera-mariabackup.nix

```diff
@@ -184,40 +184,40 @@ in {
     galera_03.wait_for_unit("mysql")
     galera_03.wait_for_open_port(3306)
     galera_02.succeed(
-        "sudo -u testuser mysql -u root -e 'use testdb; select test_id from db1;' -N | grep 37"
```

Comment:
I'd need some context here. Were these tests broken before this PR? Are these changes needed because of the version bump?

In both cases, please add the explanation to the commit message.

Reply:
MariaDB 10.4 introduces a number of changes to the authentication process, intended to make things easier and more intuitive. See https://mariadb.com/kb/en/authentication-from-mariadb-104.
Added to release notes.

Reply:
Added comment in commit.